fastapi/error_handlers.py

```diff
@@ -20,7 +22,10 @@
 def convert_exception_to_status_body(exc: Exception) -> tuple[int, dict]:
     body = {}
     status_code = status.HTTP_500_INTERNAL_SERVER_ERROR
-    details = "Internal Server Error"
+    if os.getenv("FASTAPI_DEBUG") == "1":
+        details = repr(exc)
+    else:
+        details = "Internal Server Error"
```

Comment:
```suggestion
    details = "Internal Server Error"
    if config.get_misc("fastapi", "dev_mode"):
        traceback = "".join(traceback.format_exception(*sys.exc_info()))
        details = f"{details}\n{traceback}"
```

you must also import the right modules...

```
import sys
import traceback

from odoo.tools.config import config
```

Reply:
Thanks for the suggestion. Applied with slight changes.